site/includes/most-requested-theme-topic.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="container">
+	<div class="row">
+	<div class="pddng-r-0 col-md-2">
```

Comment:
The CSS "pddng-r-0" is undefined.

Reply:
Yeah.  That was copied from the design.canada.ca page provided in the ticket.  If we don't have that entry should I add it in the provisional section?  It is a padding-right: 0px;

Reply:
I wonder if that padding 0 is needed. In a CSS bootstrap 3 grid, there is already no padding, so I think that class can be removed.

fyi - Regarding the addition of utility CSS class, we are preferring the CSS naming coming from Bootstrap 4. Like if we have a use case to set no padding to right to none, it will be something like `pr-0`.